cli/src/download_user_pool_files.rs

```diff
@@ -0,0 +1,212 @@
+use std::collections::HashSet;
+
+use crate::*;
+use serde::Serialize;
+use solana_client::{nonblocking::rpc_client::RpcClient, rpc_config::*};
+use solana_sdk::account::ReadableAccount;
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct RpcKeyedAccount {
+    pub pubkey: String,
+    pub account: UiAccount,
+}
+
+impl RpcKeyedAccount {
+    pub fn new<T: ReadableAccount>(pubkey: &Pubkey, account: &T) -> Self {
+        Self {
+            pubkey: pubkey.to_string(),
+            account: UiAccount::encode(pubkey, account, UiAccountEncoding::Base64, None, None),
+        }
+    }
+}
+
+#[derive(Parser, Debug)]
+pub struct DownloadUserPoolFilesParams {
```

Comment:
Why we need this function?

Reply:
Not needed. But putting it just for this PR to allow reviewer able to dump and run the test for v2 to v3 (dynamic position) test. Will remove it in next PR.

The test is here
https://github.com/MeteoraAg/dlmm-sdk/blob/96130c56264436c3edb9975d31b0e5b89fd9ae33/ts-client/src/test/migrate_to_dynamic_position.test.ts#L15